### PR TITLE
mades edits to the ArcGIS README

### PR DESCRIPTION
**Pull request by Tiny-Pickles**
Just added a starter tutorial to download ArcGIS Pro. Also started a basic list of other tutorial I will want to make in the future. This list might change depending on the requirements and needs for my Senior Project.

### CODE REVIEW

File: ArcGIS.md [hunk 1]
```diff
@@ -1,4 +1,28 @@
-# ArgGIS #
+# ArgGIS Tutorials #
+
+## What is GIS? ##
+A geographic information system is used to display various types of data within a map-like format. GIS can connect various types of local data that display the physical geographical locations, as well as additional descriptions. 
+## ArcGIS Online ##
+Is a cloud-based software that allows you to build interactive visual maps. This is Esri’s web-based mapping platform. You will need a License through ArcGIS in order to use it and thankful we have access to ArchGIS through Penn State!
```
Comment (am0eba-byte) [inline ArcGIS.md:6]: noice

File: ArcGIS.md [hunk 2]
```diff
@@ -1,4 +1,28 @@
...
+
+## ArcGIS Pro ##
+This is Esri’s desktop application for ArcGIS. Please keep in mind the ArcGIS was made for Windows, not Macs. 
+
+## How to Create an Account and Download ArcGIS Pro: ##
+1. Go to [ArcGIS Online](https://www.esri.com/en-us/arcgis/products/arcgis-online/overview) and click the Sign In button.
+2. You will want to sign with the ArcGIS Organization’s URL section.
+    1. You will want to type in the word pennstate
```
Comment (am0eba-byte) [inline ArcGIS.md:14]: i'd put the keyword pennstate in blockquotes so it's easier for folks to see and copy, with tickmarks like this: `pennstate`

File: ArcGIS.md [hunk 3]
```diff
@@ -1,4 +1,28 @@
...
+    2. Now this will take you to the Penn State login page. This page will prompt you to login in with your PSU account.
+3. Once you are logged into your ArcGIS account that is registered through Penn State go to the top right hand corner of your screen and click your account tab. Once the drop menu is open, click the My settings section.
+4. Then you will want to click the Licenses tab near the top left corner of your webpage.
+5. Here you will see all Licenses Penn State has to offer for any PSU students that want to use ArcGIS.
+6. Scroll down and find the ArcGIS Pro License click the download link. Once the download has completed, click and open the downloaded ArcGIS file.
```
Comment (am0eba-byte) [inline ArcGIS.md:19]: screenshots of these license screens might be helpful here too

File: ArcGIS.md [hunk 4]
```diff
@@ -1,4 +1,28 @@
...
+    1. You will have a pop-up window that will guide you through a series of steps for the installation process.
+    2. Choose the desired destination where you would like to keep your ArcGIS desktop program. Then press the Next button.
+7. After the extraction process is completed make sure the *Launch the setup program* box is checked. :heavy_check_mark: 
+8. Then press the Close button.
+## Formatting Data for ArcGIS: ##
```
Comment (am0eba-byte) [inline ArcGIS.md:24]: I'm excited to see this section!

**Review by am0eba-byte — APPROVED** [review]
This is great! nice work. merging this branch will not result in any merge conflicts with main. We will have to experiment with editing files that are also on the main branch in a feature branch, and try the `git rebase` command to see how it works once we get there!

File: ArcGIS.md (hunk 2 above)
Comment (ebeshero) [reply ArcGIS.md:14]: Agreed w/ @am0eba-byte : Using the code blocks with the `backticks` makes it super easy to see where to copy / paste.

File: ArcGIS.md [hunk 5]
```diff
@@ -1,4 +1,28 @@
...
+## How to Look up Data within Living Atlas: ##
+## Adding Data to your Map: ##
+## Display Data and Tables on your Map: ##
```
Comment (ebeshero) [inline ArcGIS.md:27]: @Tiny-Pickles These sections look just right to me so far! We may want to add more as you dig into analyzing correlations in your data sets, as in, being able to highlight which reservations are close to corporate camps, etc?

**Review by ebeshero — APPROVED** [review]

**Comment by ebeshero** [comment]
Following @am0eba-byte 's excellent suggestion, I'm going to merge this PR into main, but have @Tiny-Pickles keep your branch alive so you can keep working in it and submit new PRs as we go. We'll talk about **rebasing** your branch (to get updates from main) this evening.

**Pull request merged**